packages/app/src/modules/options.js

```diff
@@ -39,7 +39,7 @@ export const options = {
     numberType: { defaultValue: 'VALUE', requestable: false },
     showHierarchy: { defaultValue: false, requestable: true },
     legendSet: { defaultValue: undefined, requestable: false },
-    legendDisplayStrategy: { defaultValue: undefined, requestable: false },
+    legendDisplayStrategy: { defaultValue: 'FIXED', requestable: false },
```

Comment:
What does `legendDisplayStrategy` have to do with this PR? Could you add a comment in the description please

Reply:
That's the actual fix for the original problem.

The LegendSet option is a "advanced" option which uses several simple options: `legendDisplayStrategy`, `legendDisplayStyle` in order to control the value of `legendSet`.
When you choose `FIXED` for `legendDisplayStrategy` the dropdown with the available legends becomes available.